### PR TITLE
Dev 207 update GitHub actions

### PR DESCRIPTION
**Pull request by iranimij**

### CODE REVIEW

**Review by sprankhub — CHANGES_REQUESTED** [review]
Looks good to me. But please also add:

* `coding-standard.yml`
* `mess-detector.yml`
* `phpstan.yml`

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      - uses: actions/checkout@v4
+      - name: M2 Integration Tests with Magento 2 Version 2.4.6-p3 (PHP 8.2)
+        uses: extdn/github-actions-m2/magento-integration-tests/8.2@master
+        with:
+          module_name: CustomGento_Cookiebot
+          composer_name: customgento/module-cookiebot-m2
+          ce_version: '2.4.6-p3'
+          composer_version: '2'
+      - uses: actions/checkout@v4
+      - name: M2 Integration Tests with Magento 2 Version 2.4.6-p2 (PHP 8.2)
```
Comment (sprankhub) [inline .github/workflows/integration.yml:316]: Why don't we test this version with PHP 8.1? :thinking:

File: .github/workflows/integration.yml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        ports:
+          - 9200:9200
+        env:
+          'discovery.type': single-node
+          'xpack.security.enabled': false
+          ES_JAVA_OPTS: "-Xms64m -Xmx512m"
+        options: --health-cmd="curl localhost:9200/_cluster/health?wait_for_status=yellow&timeout=60s" --health-interval=10s --health-timeout=5s --health-retries=3
+    steps:
+      - uses: actions/checkout@v4
+      - name: M2 Integration Tests with Magento 2 Version 2.4.4-p6 (PHP 8.1)
```
Comment (sprankhub) [inline .github/workflows/integration.yml:26]: Why don't we test this version with PHP 7.4? 🤔

File: view/frontend/templates/script.phtml [hunk 3]
```diff
@@ -1,4 +1,4 @@
 <?php /** @var Magento\Framework\View\Element\Template $block */ ?>
 <?php /** @var CustomGento\Cookiebot\ViewModel\Script $viewModel */ ?>
 <?php $viewModel = $block->getData('view_model') ?>
-<?= $viewModel->getScript() ?>
+<?= $viewModel->getScript() // phpcs:ignore ?>
```
Comment (sprankhub) [inline view/frontend/templates/script.phtml]: Why ignore? This should get a `/* @noEscape */`.

**Review by sprankhub — CHANGES_REQUESTED** [review]

File: view/frontend/templates/script.phtml (hunk 3 above)
Comment (iranimij) [reply view/frontend/templates/script.phtml]: Since the output is static and does not need to be escaped

**Review by sprankhub — APPROVED** [review]

**Pull request merged**